scripts/exporters/export_pdf_extraction_docx.py

```
@@ -0,0 +1,199 @@
+        if isinstance(bx, (int, float)):
+            x0 = float(bx)
+
+    # Keep text slightly before image when bbox ties.
+    type_rank = 0 if ct == "text" else 1
+
+    block_id = md.get("block_id", "")
+    block_num = 0
+    if isinstance(block_id, str) and block_id.startswith("T"):
+        try:
+            block_num = int(block_id[1:])
+        except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)